doc/source/release-notes/Percona-XtraDB-Cluster-5.6.40-26.25.rst

```diff
@@ -0,0 +1,45 @@
+.. rn:: 5.6.40-26.25
+
+===================================
+Percona XtraDB Cluster 5.6.40-26.25
+===================================
+
+Percona is glad to announce the release of
+Percona XtraDB Cluster 5.6.40-26.25 on June 20, 2018.
+Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/>`_
+or from our :ref:`software repositories <installation>`.
+
+Percona XtraDB Cluster 5.6.40-26.25 is now the current release,
+based on the following:
+
+* `Percona Server 5.6.40 <https://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.40-84.0.html>`_
+
+* Galera/Codership WSREP API Release 5.6.40
+
+* Galera Replication library 3.23
+
+All Percona software is open-source and free.
+
+Improvements
+============
+
+* :jirabug:`PXC-907`: Added :variable:`wsrep_RSU_commit_timeout` variable to
+  configure RSU wait for active commit connection timeout (in micro-seconds).
```

Comment:
Please add complete description to wsrep-system-index.rst. This will also fix DOC-747 (which is blocker for this issue).